src/GUI/GUI.java

```diff
@@ -43,7 +43,8 @@ public class GUI {
     private static JLabel outputLabel = new JLabel("", SwingConstants.CENTER);
     private static JLabel headlineLabel = new JLabel("Währungsrechner");
     private static JLabel authorLabel = new JLabel(VERSION + " by Leon, Jonas, Ewin");
-    private static JLabel settingsLblBtn = new JLabel(new ImageIcon("resources/buttons/button_loading.gif"));
+    private static JLabel settingsLblBtn = new JLabel(new ImageIcon("resources/buttons/settings_button.png"));
```

Comment:
Also wir haben scheinbar das image icon hier doppelt weil es nochmal in der "drawSettingsBtn" funktion auftaucht, als neue variable? Schau mal bitte wie wir das bei anderen icons machen und dann würd ich sagen machen wir das icon setzen genau wie da. Also wenn icon setzen hier oben passiert dann auch hier oben, und wenn icon setzen immer in der Funktion gemacht wird würd ich das bei dem settings label dann auch so machen.

Aber danke dass dir das aufgefallen ist!